userguide/tn_vcenter.rst

```diff
@@ -4,12 +4,27 @@
 vCenter Plugin
 ==============
 
-The %brand% vCenter plugin integrates the control and operation of
-%brand% into
-`VMware vCenter Server
-<https://www.vmware.com/products/vcenter-server.html>`__.
-
-The %brand% vCenter plugin is available separately from %brand%. Please
-contact the iXsystems Support team at :literal:`support@iXsystems.com`
-or :literal:`1 (408) 943-4100 option 2` to request the plugin and
-documentation.
+`vCenter Server <https://www.vmware.com/products/vcenter-server.html>`__
+is server management software that uses a single console to manage a
+virtual infrastructure across a hybrid cloud of physical and virtual
+machines. The %brand% vCenter plugin integrates the control and
```

Comment:
Still not sure about "integrates".  Maybe
"makes it possible to provision and use %brand% storage from within vCenter Server."?